lib/tasks/generate_erd.rake

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/LineLength
+def generate_erd
+  system 'erd --inheritance --filetype=dot --direct --attributes=foreign_keys,content'
```

Comment:
Can you try changing this to `erd --inheritance ...`? Then maybe we can remove the LineLength violations.